tests/integration/device.js

```diff
@@ -0,0 +1,26 @@
+
+const viewports = {
+  mobile: {
+    width: 400,
+    height: 800,
+    hasTouch: true,
```

Comment:
have you tried the `isMobile` option ? This might have a serious impact on our rendering IIUC https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#puppeteerconnectoptions

Reply:
I don't think this will change anything for now but clearly I should have set it and forgot. Thanks!